templates/etc/systemd/system/prometheus.service.j2

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Prometheus
+After=network.target
+
+[Service]
+Type=simple
```

Comment:
I did't find information about forking main process. I think simple type is normal.